### PR TITLE
Add RawTriangular fast path to Solve and delete SolveTri

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

**Comment by btracey** [comment]
#232  opened about the changed tolerances.

**Comment by btracey** [comment]
Fixes #211

File: mat64/solve.go [hunk 1]
```diff
@@ -21,8 +27,35 @@ func (m *Dense) Solve(a, b Matrix) error {
...
+	case RawTriangular:
+		side := blas.Left
+		tA := blas.NoTrans
+		if aTrans {
+			tA = blas.Trans
+		}
+		if m != bMat {
+			m.Copy(b)
+		} else if bTrans {
+			tmp := DenseCopyOf(b)
```
Comment (kortschak) [inline mat64/solve.go]: Add a comment here.

Comment (btracey) [inline mat64/solve.go]: What would you like the comment to say? Just that the values are shadowed?

Comment (kortschak) [inline mat64/solve.go]: Yeah.

Comment (btracey) [inline mat64/solve.go]: Done.

**Comment by btracey** [comment]
PTAL

**Comment by kortschak** [comment]
LGTM

**Pull request merged**